javascript-data-model/order-history.js

```diff
@@ -0,0 +1,65 @@
+var orderHistory = [
+  {
+    orderNumber: '114-3941689-8772232',
+    totalNumOfProducts: 1,
+    itemInOrder: [0],
+    productName: 'JavaScript for impatient programmers',
+    dateOrderPlaced: 'August 4, 2020',
+    productType: 'Literature',
+    productPrice: 31.55,
+    dateOrderDelivered: 'August 8, 2020',
+    author: true,
+    authorName: 'Rauschmayer, Dr. Axel',
+    totalPrice: 34.00
+  },
+  {
+    orderNumber: '113-9984268-1280257',
+    totalNumOfProducts: 1,
+    itemInOrder: [0],
+    productName: 'The Timeless Way of Building',
+    dateOrderPlaced: 'July 19, 2020',
+    productType: 'Literature',
+    productPrice: 41.33,
+    dateOrderDelivered: 'July 20, 2020',
+    author: true,
+    authorName: 'Alexander Christopher',
+    totalPrice: 44.53
+  },
+  {
+    orderNumber: '114-2875557-9059409',
+    totalNumOfProducts: 1,
+    itemInOrder: [0],
+    productName: 'Gamecube Controler Adapter. Super Smash bros Switch Gamecube adapter for WII U, PC. Support Turbo and Vibration Features. No Drive and No Lag-Gamecube Adapter',
+    dateOrderPlaced: 'July 4, 2020',
+    productType: 'Hardware',
+    productPrice: 15.98,
+    dateOrderDelivered: 'July 7, 2020',
+    author: false,
+    authorName: null,
+    totalPrice: 15.98
+  },
+  {
+    orderNumber: '113-2883177-2648248',
+    totalNumOfProducts: 2,
+    itemInOrder: [
+      {
+        productName: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+        productType: 'Hardware',
+        productPrice: 94.95,
+        author: false,
+        authorName: null
+      },
+      {
+        productName: 'The Art of Sql',
+        productType: 'Literature',
+        productPrice: 33.99,
+        author: true,
+        authorName: 'Faroult, Stephanie'
+      }
+    ],
```

Comment:
In this order, this property holds an array of objects, but in all the other orders, it holds an array with a `0` in it. That is not a consistent format, and will lead to bugs